scripts/docker/env/production/celery.env

```diff
@@ -39,4 +39,4 @@ MAX_DOCUMENT_SIZE=2
 # GEOSERVER_ADMIN_PASSWORD=admin
 # See https://github.com/geosolutions-it/geonode-generic/issues/28
 # to see why we force API version to 1.24
-DOCKER_API_VERSION="1.24"
+DOCKER_API_VERSION="1.24"
```

Comment:
I suppose that there is something wrong with your editor :)

Reply:
oh, I use Sublime on Windows. This file does not need any changes